test/fixtures/products.yml

```
@@ -4,14 +4,14 @@ apple:
   photo_url: just a string
   description: red apple 
   stock: 5
-  merchant_id: merchant_one.id
+  merchant_id: 1
```

In fixture files, we can also set relations like this! 

```suggestion
  merchant: merchant_one
```

Note: We can still access product.merchant_id, which will be equal to merchant_one.id in this case.